Encounter1/MainActivity.cs

```diff
@@ -80,7 +80,7 @@ public string CreateDB()
         {
             var output = "";
             output += "Creating Database if it doesn't exists";
-            string dpPath = System.IO.Path.Combine(System.Environment.GetFolderPath(System.Environment.SpecialFolder.Personal), "users.db3"); 
```

Comment:
It might be useful to move file names to separate class for constants in order to make changes easier